.github/workflows/ci.yml

```diff
@@ -52,3 +54,6 @@ jobs:
         run: npm install
       - name: Start Tests
         run: npm run mocha_test -- -g ${{ matrix.mcVersion }}
+
+    env:
+      RUN_EXCLUDED_TESTS: ${{ github.event.schedule == '0 0 * * *' }}
```

Comment:
Let's make it run every hour to make sure it works, then we switch it back to every day